main/src/cgeo/geocaching/maps/PositionHistory.java

```diff
@@ -1,14 +1,14 @@
 package cgeo.geocaching.maps;
 
 import cgeo.geocaching.storage.DataStore;
-import cgeo.geocaching.utils.AndroidRxUtils;
 
 import android.location.Location;
 
 import java.util.ArrayList;
 
 import io.reactivex.schedulers.Schedulers;
 
+
```

Comment:
Can you remove the new newline?

Reply:
which one - before or after line 10new? (line 12new is a comment belonging to the class definition)

Reply:
done